site/app/templates/admin/users/RotatingSectionsForm.twig

```diff
@@ -9,12 +9,18 @@
             else {
 				$('#redo-data').show();
             }
+            if (val != "fewest"){
```

Comment:
You would want a `!==` here for strict inequality
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_inequality

Normally the JS linter would catch this but it can't work on JS embedded inside html/twig pages